### PR TITLE
chineses readme

### LINKED ISSUE #235: 添加 init 模板
https://github.com/vangie/poetry
https://github.com/vangie/packed-selenium-java-example
https://github.com/vangie/couchbase-example

https://github.com/muxiangqiu/fc-express-nodejs8
https://github.com/muxiangqiu/fc-next-nodejs8
https://github.com/muxiangqiu/fc-skeleton-nodejs8

### PR DESCRIPTION
**Pull request by coco-super**
fix #235

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Fun 作为一个命令行工具，内置了多个子命令，比如 config、loc
       - [开发函数计算的正确姿势——支持 ES6 语法和 webpack 压缩](https://yq.aliyun.com/articles/701714)：帮助快速搭建一个函数计算 nodejs 项目骨架，支持 es6 代码编译成 es5。
       - [函数计算实现 oss 上传较小 zip 压缩文件的自动解压](https://github.com/coco-super/simple-fc-uncompress-service-for-oss)：快速搭建 oss 上传较小 zip 压缩文件自动解压（简单法）的模板项目。
       - [函数计算实现 oss 上传较大 zip 压缩文件的自动解压](https://github.com/coco-super/streaming-fc-uncompress-service-for-oss)：快速搭建 oss 上传超大 zip 压缩文件自动解压（流式法）的模板项目。
+      - [五言绝句古诗的模板](https://github.com/vangie/poetry)：介绍把一个 TensorFlow CharRNN 训练的自动写五言绝句古诗的模型部署到函数计算。
```
Comment (tanhe123) [inline README.md]: 函数计算 TensorFlow CharRNN 五言绝句古诗模板

Comment (coco-super) [reply README.md]: OK

Comment (tanhe123) [inline README.md]: Template project for writing Five-Character Verse using TensorFlow CharRNN

File: README.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ Fun 作为一个命令行工具，内置了多个子命令，比如 config、loc
...
+      - [函数计算 selenium chrome java 模板项目](https://github.com/vangie/packed-selenium-java-example)：一个使用 brotli 算法压缩的 selenium chrome java 模板项目。
+      - [函数计算 couchbase 模板项目](https://github.com/vangie/couchbase-example)：该项目模板是一个在阿里云函数计算平台 nodejs8 环境下安装并配置 couchbase 的模板项目。
```
Comment (tanhe123) [inline README.md]: Template project for installing and configuring couchbase on function compute.

File: README.md [hunk 3]
```diff
@@ -33,6 +33,12 @@ Fun 作为一个命令行工具，内置了多个子命令，比如 config、loc
...
+      - [express 项目接入函数计算](https://github.com/muxiangqiu/fc-express-nodejs8)：通过该模板，可以快速将 express 项目接入函数计算。
```
Comment (tanhe123) [inline README.md]: Help quickly build a express project of Function-Compute.

File: README.md [hunk 4]
```diff
@@ -33,6 +33,12 @@ Fun 作为一个命令行工具，内置了多个子命令，比如 config、loc
...
+      - [nextjs 应用接入到函数计算](https://github.com/muxiangqiu/fc-next-nodejs8)：提供了一个 fun 模块，通过该模板，三分钟就可以让 next.js 应用在函数计算中运行起来。
```
Comment (tanhe123) [inline README.md]: Help quickly build a next.js project of Function-Compute.

File: README.md [hunk 5]
```diff
@@ -33,6 +33,12 @@ Fun 作为一个命令行工具，内置了多个子命令，比如 config、loc
...
+      - [es6 代码编译成 es5 项目模板](https://github.com/muxiangqiu/fc-skeleton-nodejs8)：提供一个 fun 模板，通过 webpack 将 es6 代码编译成 es5，并且剪切打包压缩成一个 js 文件，然后将该 js 文件上传到函数计算中运行。
```
Comment (tanhe123) [inline README.md]: Help quickly build a webpack project of Function-Compute in order to compile es6 to es5.

**Pull request merged**